sdk/tests/conformance/glsl/misc/non-ascii.vert.html

```diff
@@ -18,18 +18,35 @@
 <body>
 <div id="description"></div>
 <div id="console"></div>
-<script id="vertexShader" type="text/something-not-javascript">
+<script id="vertexShader" type="text/plain">
+Loading...
+</script>
+<script>
+"use strict";
+
+// Via `[].map.call(str, x => x.codePointAt(0));`
+let a = [8218,177,8218,234,8218,205,8218,96,8218,114,8218,98,8218,104,8218,104,8218,197,8218,205,
+         8218,200,8218,162,8218,197,8218,183,129,66];
+let b = [8218,115,8218,710,8218,8240,8218,8220,129,64,8218,104,8218,8220,129,64,8218,109,8218,143,
+         8218,8221,129,64,8218,96,8218,114,8218,98,8218,104,8218,104];
+a = String.fromCodePoint(...a);
+b = String.fromCodePoint(...b);
+
+// -
+
+vertexShader.textContent = `
 // Non ascii data in source should fail
 // See GLSL ES Spec 1.0.17 section 3.1 and 3.2
-// ����͂`�r�b�h�h�ł͂Ȃ��ł��B
-// �s�������@�h���@�m�����@�`�r�b�h�h
-uniform mat4 �m�����`�r�b�h�h;
+// ${a}
+// ${b}
+uniform mat4 ${b};
```

Comment:
This identifier's changed from the original test. The comment has spaces but the uniform's name didn't. Would you consider restoring this to the original value?

Reply:
You know, this is the first time I've seen it render as ~"NotASCII". I've only ever seen garbage elsewhere. :S